Source/Wit/Private/Voice/Matcher/VoiceIntentWithEntitiesMatcher.cpp

```diff
@@ -75,11 +76,12 @@ void UVoiceIntentWithEntitiesMatcher::OnWitResponse(const bool bIsSuccessful, co
 		else
 		{
 			MatchingEntities.Add(*MatchingEntity);
+			MatchingEntitiesNum++;
 			MatchingValues.Add(MatchingEntity->Value);
 		}
 	}
 
-	const bool bIsAllRequiredEntitiesMatched = MatchingEntities.Num() == RequiredEntityNames.Num();
```

Comment:
see line 71, MatchingEntities never be empty

Reply:
Good catch. This check would have been before we added the empty entities in a non-match case